src/main/java/com/commercetools/sync/commons/BaseSync.java

```diff
@@ -24,7 +24,7 @@ protected BaseSync(@Nonnull final U statistics, @Nonnull final V syncOptions) {
      * actions on the existing resource if it exists or create it if it doesn't.
      *
      * @param resourceDrafts the list of new resources as drafts.
-     * @return an instance of {@link CompletionStage}&lt;{@code U}&gt; which contains as a result an instance of
+     * @return an instance of {@link CompletionStage}&lt;{@code U}&gt; which contains as a result of an instance of
```

Comment:
`which contains as a result of an instance of` -> `which contains, as a result, an instance of`